docs/release-notes/self-hosted/self-hosted-v14.0.0.md

```diff
@@ -0,0 +1,87 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v14.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.7
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.12
+---
+
+# Self-hosted v14.0.0
+
+These release notes are for [Codacy Self-hosted v14.0.0](https://github.com/codacy/chart/releases/tag/14.0.0){: target="_blank"}, released on June 20, 2024.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v14.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v14.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:
+
+    -   [Codacy Analysis CLI 7.9.5](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.9.5)
+    -   [Codacy Coverage Reporter 13.10.15](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.10.15)
```

Comment:
@Ngwerume I know you already validated and you said "the newer version do not work, due to the nature of updates we are doing on SH which do not update the API."... 🙏 

But I still don't understand this downgrade: on the previous [Self-hosted v13.0.0](https://docs.codacy.com/release-notes/self-hosted/self-hosted-v13.0.0/) we used [Codacy Coverage Reporter 13.13.14](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.13.14).
Shouldn't we keep this same version? 🤔

Reply:
We could add a note. Something like: `Codacy is downgrading to Codacy Coverage Reporter version 13.10.15. Versions above 13.10.15 are currently incompatible with recent updates.`

@Ngwerume @claudiacarpinteiro Thoughts?

Reply:
@claudiacarpinteiro we ran into issues with 13.13.14 and resolved with the affected customers, the open end here was we did not clarify with you to change the documents accordingly. 
@nicklem We can change the wording to something like Codacy Coverage Reporter for Self Hosted works with version 13.10.15 but I am ok with how you want to present this.
All SH from v12 should be using the same coverage reporter as we did not upgrade the APIs only changes to tools and k8s bumps.  As we did not update the API, the never version of coverage reporter fails as the API is not available